components/automate-ui/src/app/components/projects-dropdown/projects-dropdown.component.html

```diff
@@ -9,7 +9,8 @@
     [disabled]="disabled"
     (keydown.arrowup)="moveFocus($event)"
     (keydown.arrowdown)="moveFocus($event)">
-    Select projects... <chef-icon aria-hidden>keyboard_arrow_down</chef-icon>
+    <div id="projects-selected">{{ label }}</div>
+    <chef-icon aria-hidden>keyboard_arrow_down</chef-icon>
   </button>
   <chef-click-outside omit="column-dropdown-toggle" (clickOutside)="closeColumnDropdown()">
```

Comment:
Cut-and-paste residual: 
1. Suggest rename to just `closeDropdown` -- there is no "column" lurking around here.
2. Remove the omit property entirely; it is referencing a class that does not exist.
```suggestion
  <chef-click-outside (clickOutside)="closeDropdown()">
```